### ISSUE TITLE
Update branding

### ISSUE THREAD

**Issue by Piedone** [OrchardCMS/OrchardCore#6279]
We don't really have concise branding guidelines or even proper branding assets, just what's left over from a long time ago: https://docs.orchardcore.net/projects/O1/en/latest/Documentation/Walkthroughs/#orchard-branding

What I'd imagine:

- Possibly retouch the existing design a bit: E.g. the current logo's outer circle is strangely uneven. 
- Maybe use a different branding color than the green there or just use those but then we should settle with the same main brand colors everywhere. Currently, there are these old greens and greys, then there's the other green on the [website](https://www.orchardcore.net/), a blue [on the docs site](https://docs.orchardcore.net/en/dev/) and another blue for the social preview: 
![image](https://user-images.githubusercontent.com/1976647/82947994-3e2e6380-9fa1-11ea-82f7-16942d21e49f.png)
- Create new, high-resolution bitmaps.
- Possibly some branding guidelines on how to use the new assets.

We can potentially sponsor designers for this (the bottleneck is their capacity).

[Here're the old branding assets](https://docs.orchardcore.net/en/dev/docs/resources/branding/) in the docs.

**Comment by hishamco** [comment]
@Piedone is this the official logo, or we design a new one

**Comment by Piedone** [comment]
I'd keep the central "leaves in circle" motive, but probably update it somewhat. I'd prefer not to have a completely new one though.

**Comment by parisnoble** [comment]
Hi everyone,

My name is Paris Noble, brand strategist and co-founder of Elevate Strategy & Design. For the past 2 years we've been working closely together with Lombiq Technologies on many Orchard related projects that either needed something to do with strategy and/or design. 

A few weeks ago me and @Piedone started chatting about the possibility of rebranding Orchard, which Lombiq would sponsor. 
However, to do this we would want to involve other stakeholders into the rebranding process. This is mainly because the more deeply we involve the most important Orchard stakeholders the merrier, but also because this open-source project cannot be authentically rebranded without the OS mindset.

**So what is branding and why rebranding?** 
A brand is a product, company, or service with a soul that is attractive and smart. A soul like this will bring in customers, and creates customer loyalty, and helps the longterm vision of product design. 
In other words a brand is the customer's gut feeling of any given product, company, or service — and the better the customers perceive a brand, the more successful its company will become.

The reason why companies use branding is to radically differentiate themselves from the competition and create customer loyalty. Also, it may become the most valuable asset of a company — since as humans we make decisions based on emotions than on logic (yes, even developers :)), and good branding will help make stronger emotional ties between the company and its customers, followers.
Competition may steal your staff, your know-how, copy your product, beat you on price, but can never steal one's brand.

In the past two years working closely with Lombiq Technologies, we've learned how strong the Orchard community is, how much potential Orchard has, and how vital it is to sharpen its competitive edge. 

**What are the steps of branding?**
Branding is much more than a logo or brand identity. Therefore first we need to establish a strategy. Brand strategy basically bridges the gap between business strategy and visual design. Therefore it's good to have good knowledge about the project we wish to brand. 
1. The first step is usually setting a brand platform, which is usually done by the founders and key stakeholders via one or more workshops. The brand platform sets the most foundational parts of the new brand, including its personality, voice and tone, key messages it want to convey, and the overall most important strategy based on longterm goals and business strategy.
2. Creating the visual identity. This is the part where we write a brief based on the project goal and established brand strategy. If the project goal is a new visual identity or in this case a rebranding, then this is the step when we start developing the new logo etc. 
3. Brand atmosphere & digital design: This is the part when we start writing copy, put out key messaging, create content, develop products etc for the new brand. Also this is the part when we can start creating new websites, digital products for our product, company or service. 

**So why do we want to start rebranding Orchard?**
Branding is super important to make Orchard a widely known, competitive product, to truly make it an industry sensation. And with its amazing community and technology behind it, I am sure it can once become something disruptive.

What do you guys think? Would you be open to rebrand Orchard from the ground up and create a brighter, bolder, more competitive future we can all be a part of?

**Comment by agriffard** [comment]
My first remark would be that I don't see Orchard as a Product.

Here are some other thoughts:
Orchard might be underserved by the fact that you can do a lot of different kind of apps from a simplest to a more complex one so people can be confused by all those possibilities and the actual power of the framework.
Orchard is often perceived as a project with a steep learning curve. Even when some bloggers write articles, it is hard to go beyond the level 101.

In order to spread on a larger scale, I would say that Orchard might need:
- Direction: What is the current status, the next steps, ...
- Explanations: It must be clear after an hour of searching/testing if Orchard fits your needs.
- Recognition: State that every contribution counts. 
- Transparency: Do not presume that, because this is an open source project, people understand exactly how everything works.
- Visibility: Spread the word, share about the true potential.
	
I am always impressed by the dedication of the contributors, the amount of time they can spend to improve the project.

We should also ask ourselves 'What makes a Framework/CMS popular?'

I hope that a new branding will be able help but it would be just one step on the path to the success we are all expecting.

**Comment by agriffard** [comment]
This reminds me a story about how the potaoes have been widespread in France:
https://commoninfirmities.com/2017/02/03/who-tricked-whom-into-eating-potatoes/

TLDR: If you want to make something successful, don't give it for free, people will not want it. 
Instead, claim that it is strictly forbidden to use it but leave it available by some way, people will be intrigued and will want to try it.
This is some kind of reverse psychology, like the forbidden fruit.

**Comment by Skrypt** [comment]
I don't mind to be honest. This remembers me a story in 01 where guys wanted to rebrand the https://www.orchardcore.net/ website but it never concluded. Here, we have to be prudent into deciding what kind of rebranding it would be.

Changing the CMS name? That would be a bold move to be honest. The issue being that there is a lot of content already published on social medias and youtube using the Orchard name.

If the goal is to promote Orchard better then I'd say that it starts by making the current https://www.orchardcore.net/ more appealing and to make it usefull in a way where people will actually use it to promote their websites or services. Currently, I barely use the https://www.orchardcore.net/ website apart from clicking on the Podcast meeting link on the frontpage. Also, if we look at other platforms. Let's take https://ghost.org/ for example, they also offer SaaS services related with their CMS. Here with Orchard, there is no product offering of that kind, and no real budget for promoting the framework since it's just a common tool used by everyone. Also, Lombiq, promotes itself as a company that offers SaaS services around Orchard Core since version 01 so maybe this is where the rebranding money should be spent since it's a "company product". Just some thoughts ... to continue 😄

**Comment by Piedone** [comment]
[Here're the old branding assets](https://docs.orchardcore.net/en/dev/docs/resources/branding/) in the docs.

So as we discussed in today's meeting the first step would be to produce a variety of logos that we can properly use in a variety of places and make Orchard logo usage consistent across everywhere. This is more of a technical step which is really necessary because currently there's no easy way to get proper Orchard logos (like if you want to use them in a presentation).

And in the meantime, we can do the long discussion about what needs to be adjusted in Orchard's communication (like the contents of the website), if any. Quite a while ago @stevetayloruk did some work on revamping the Orchard (1) website but I don't think those materials are available anymore as Steve's blog disappeared. Steve, if you're out there, please respond.

**Comment by stevetayloruk** [comment]
@Piedone what do you need from me? The blog is still up, albeit not updated for a while!

https://www.stevetaylor.me.uk/

**Comment by Skrypt** [comment]
I think showing at least a video of the OC admin dashboard unique features would be a good start. Something that would summarize what is Orchard Core and that shows it.

**Comment by Piedone** [comment]
Ah thanks @stevetayloruk, needed www :D. This is what I was looking for: https://www.stevetaylor.me.uk/the-new-orchard-cms-website/ (cc @parisnoble).

@Skrypt we've started working on an interactive walkthrough a while ago (but then put it on hold due to other priorities), similar to the one you can see on the demo sites here: https://tryorchard.net/ An alternative?

**Comment by parisnoble** [comment]
Hi everyone,

We've put together a high-level, more abstract concept regarding the visual rebranding of Orchard. It also lists the biggest issues with the current visual identity. 

Please note, this is only a visual rebranding concept.

Please take a look at it, and let us know if you have any ideas, comments, questions regarding it.

[Orchard core_rebranding (1).pdf](https://github.com/OrchardCMS/OrchardCore/files/5982338/Orchard.core_rebranding.1.pdf)

**Comment by Piedone** [comment]
Sounds good to me. BTW I don't think a simplification like in Basecamp's case is necessary since the Orchard logo is already simple as it is. (However, I agree that it has inconsistencies within, and how we use it, etc. hence why we have this discussion. Just adding that we're more like adding order and consistency here rather than reimagining a brand.)

**Comment by parisnoble** [comment]
Does anyone else have any comments on the Orchard Core rebranding PDF?

**Comment by Skrypt** [comment]
I agree with the changes here. The logo has been used in different forms over the years and making it.

![image](https://user-images.githubusercontent.com/3228637/108744157-9cc04800-7507-11eb-9f2c-f55cb2fe7f6f.png)

This seems to me appropriate. I don't understand though the leaf on it's right side. Maybe just an example of a leaf shape standing by it's own. Here, one thing that could be tried is to use color or to bring some tri-dimensional aspect to the logo shape because that's what people have tried over the years and I think that's why that circle has been added over time. Else, to me the 3 leaves + steam is the actual logo.

**Comment by Skrypt** [comment]
Of course the goal here is to standardize the logo shapes using a vectorial image that can be scaled and that won't be altered based on the size it's printed. The proportions will be kept and the logo will be identical for everyone.

I like the fact that it gives more importance to the leafs + stem in this new logo.

**Comment by parisnoble** [comment]
@Skrypt 

Exactly!

We will be sending some draft versions of the logo update soon!

**Comment by parisnoble** [comment]
Hi everyone,

We've made three draft versions for the Orchard Core logo update.

Draft 1, 2, and 3 can be combined with either typography A or B.

Please let us know which logo update meets most the future of Orchard Core, which one shall we proceed with? (Choose one)
Which typography do you prefer? (Choose one)
What do you think about the colors, do they match Orchard's long-term brand goals? (this is one color palette)
Which version with which typography combined shall we proceed? (i.e. 3B or 1A etc)
Any other feedback or suggestions?

After we have decided on how and with which option shall we proceed now, we shall come back with a finalized (or close to finalized) version of the logo. After that, there is a possibility of having one more feedback/review/finalizing round until the final logo is made. 

[Orchard core_rebranding (2).pdf](https://github.com/OrchardCMS/OrchardCore/files/6092990/Orchard.core_rebranding.2.pdf)

**Comment by Piedone** [comment]
1. Please let us know which logo update meets most the future of Orchard Core, which one shall we proceed with? (Choose one) 1.
2. Which typography do you prefer? (Choose one) A but B has a curious tech-y feeling to it.
3. What do you think about the colors, do they match Orchard's long-term brand goals? (this is one color palette) Can you show what these affect, how would they be used?
4. Which version with which typography combined shall we proceed? (i.e. 3B or 1A etc) Redundant questions ;)
5. Any other feedback or suggestions? I still think the logo's outer circle is being uneven (varying thickness) is strange. Probably originally there was some idea about a 3D effect that was later lost but I'd make it regular.

**Comment by Skrypt** [comment]
need some time to think about it. So far, first impression is that I like the first 2.

**Comment by parisnoble** [comment]
Hi there, 

Thanks for the feedback.

We shall wait for @Skrypt to think about it. If no feedback is given by Thursday, we shall proceed according to @Piedone's suggestions (including showing the color palette in action). 

Regarding the display font, I would totally suggest proceeding with B, Teko. It really does have a more distinctive techy feel to it, and is way less used than Oswald.

**Comment by Skrypt** [comment]
1 B would be me choice, but I have nothing against the second logo. This could be refreshing and making a distinction between the old CMS and the new one ...

**Comment by parisnoble** [comment]
Second logo is a more "radical update" compared to the first one. However, it still bears the essential features of the original symbol.

@Piedone & @Skrypt what do you think? Is Orchard community willing to step forward with a more radical update? Or shall we stick to version 1?

Can we agree that typography B (Teko) will be used then?

**Comment by Piedone** [comment]
Please present these at the next Tuesday's meeting.

**Comment by parisnoble** [comment]
Ok

**Comment by parisnoble** [comment]
Hi Everyone,

Thanks for yesterday's meeting and the nice feedback.

We've created a one-page slide with all 4 options.

**Goal:** We would like to agree latest by Sunday (28th) on which of the following options shall we proceed with and finalize.
**Question:** Which one of the following 4 options is most likely to best serve Orchard's long-term purpose? (i.e.: 2B)

**Please note that these versions are currently only drafts.** After we have decided on how and with which option shall we proceed now, we shall come back with a finalized (or close to finalized) version of the logo. After that, there is a possibility of having one more feedback/review/finalizing round until the final logo is made.

Looking forward to the outcome of this!

![Orchard core_rebranding_2](https://user-images.githubusercontent.com/64225612/112325546-13da2f00-8cb4-11eb-80f1-0aa85716883d.png)

**Comment by Piedone** [comment]
1A but again keep in mind that the logo's outer circle is uneven that we need to fix.

**Comment by Skrypt** [comment]
2B for a change 😉

**Comment by Piedone** [comment]
Added a simple poll for this: https://twitter.com/Lombiq/status/1375420046118432773

**Comment by hishamco** [comment]
2A ;)

**Comment by urbanit** [comment]
2A

**Comment by 0liver** [comment]
I would love to see a different color, e.g. #2fb1b1 which would like this on the homepage:

![image](https://user-images.githubusercontent.com/164360/112812877-e4de0780-907d-11eb-897c-fc589ed162d8.png)

The turquoise colors look a bit nobler to me and make the associated product more desirable than a simple green, which may appear dull or even a bit boring.

**Comment by 0liver** [comment]
Regarding the font, I find the B font a bit too aggressive with the sharp edges - I would use something a bit calmer, rounder. A is okay, but I'd use something different, still. Also, I guess we might want to look at an ensemble of fonts for use in logo, headers, and text bodies, because those three will need to go well together. Deciding on only the icon font might make the choice of good, readable header + body fonts harder down the road.

**Comment by parisnoble** [comment]
Hi @0liver ,

Thanks for the feedback. The final colors will be decided once the symbol and fonts are decided on. 

The fonts in the logo are only display fonts exclusively used for the logo. 

The font mentioned earlier in the PDF is the old font which we are changing to either A or B. And of course, there will be a secondary font family for body text and headers, etc in bold, regular, italic, etc at a later stage in the design process. But at the moment we are only deciding on whether to go with option 1A, 2A, 1B, or 2B. 

Also, as I have mentioned before the versions above are only drafts at the moment and there will be further changes to one of the voted versions above.  I.e. @Piedone 's suggestion to make the circle around the symbol a perfect circle etc.

**Comment by parisnoble** [comment]
Hey there,

It looks like 2A is the winner.

We are going to move forward with that version then, getting one step closer to finalizing the update OC logo!

Thank you everyone for the feedback and votes.

https://twitter.com/Lombiq/status/1375420046118432773

<blockquote class="twitter-tweet"><p lang="en" dir="ltr">Which one of the new <a href="https://twitter.com/hashtag/OrchardCore?src=hash&amp;ref_src=twsrc%5Etfw">#OrchardCore</a> logo proposals do you like most? See: <a href="https://t.co/PhA8gt5GR2">https://t.co/PhA8gt5GR2</a></p>&mdash; Lombiq Technologies (@Lombiq) <a href="https://twitter.com/Lombiq/status/1375420046118432773?ref_src=twsrc%5Etfw">March 26, 2021</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>

**Comment by parisnoble** [comment]
Hi Everyone,

We've put together a new version. 

- We've made a perfect circle version according to @Piedone 's suggestions. However, we think that the "offbeat" circle gives more character to the logo. Still, it's up to the community to decide!
- We've done changes according to @0liver 's suggestions for a different color (# 2fb1b1) and font.

Let us know what you think!

All the best,
Paris
[Orchard core_rebranding_3.pdf](https://github.com/OrchardCMS/OrchardCore/files/6334273/Orchard.core_rebranding_3.pdf)

**Comment by 0liver** [comment]
Form 2 - Color 2 - Type 1

**Comment by 0liver** [comment]
GitHub actually has an app for Polls: https://github.com/apps/polls

But a project administrator would have to install that. @sebastienros ?

**Comment by 0liver** [comment]
I've created a visual survey for this here: https://freeonlinesurveys.com/s/EjQ5moP4

Go vote!

**Comment by hishamco** [comment]
Form 2 - Color 2 - Type 1 As well ;)

**Comment by Skrypt** [comment]
1A, 2A, 3B

**Comment by parisnoble** [comment]
Hey Everyone!

We shall be ending the survey by Wednesday, 28th. 

Please vote if you haven't. 

here is the link again: https://freeonlinesurveys.com/s/EjQ5moP4

Secret info: we have a draw at the fonts. 50-50% is the current result for narrow and round, but we can only go with one. 

Have a great week!

**Comment by OrchardSkills** [comment]
![OrchardCore](https://user-images.githubusercontent.com/59172485/116896890-e907c280-abf1-11eb-9238-1a95d3938353.png)

**Comment by Piedone** [comment]
We're past that one (see the original post) now.

**Comment by parisnoble** [comment]
Hi everyone,

Here's is how we are going to proceed (according to the results of the survey):
- logo with regular circle
- green for color (shown in our pdf)
- narrow display font (Orchard)

We have stepped into the final steps of the Orchard rebranding and shall be posting you with the updates soon!

**Comment by sebastienros** [comment]
You mean this?

![image](https://user-images.githubusercontent.com/1165805/117060978-7a506500-acd6-11eb-9a7f-b87e74210b42.png)

**Comment by parisnoble** [comment]
Hi everyone,

As @sebastienros commented above, that is the version.

Please find attached the final version of the rebranding. 

If all is good, then we shall export all the files for the community, and the new logo, colors, and fonts are ready to use!

So please let us know if everything is ok!

[Orchard core_rebranding_4.pdf](https://github.com/OrchardCMS/OrchardCore/files/6435083/Orchard.core_rebranding_4.pdf)

**Comment by Piedone** [comment]
Looks good to me!

**Comment by sebastienros** [comment]
We should have this document available in a branding folder, including other reusable assets (vectorized and not). The a docs page that references the files. then we can start updating the sites and templates.

@parisnoble can you create a PR with the files? Or share it with someone who would do it?

**Comment by Piedone** [comment]
I'd open a PR to update the branding info in the docs once we have this finalized.

**Comment by parisnoble** [comment]
We shall send @Piedone all the files (vectorized, jpeg, monochrome, fonts, etc) to upload them and after that, they will be all ready to use!

**Comment by 0liver** [comment]
Thank you, Paris, for all your hard work to give Orchard Core a new, unified face!

As you mentioned in [this comment](https://github.com/OrchardCMS/OrchardCore/issues/6279#issuecomment-827843427), the decision between the two fonts was very tight. It was for me, too - neither of the two proposed ones really resonate with me. The one that's been chosen now looks too narrow in my eyes, especially in combination with the round logo. The other font was too round, on the other hand.

I would love to see another round of font choices. We have the logo, we have the color - let's match them up with a handful of different fonts for the community to choose from! I really think the (almost) draw between the two proposed fonts stems from people really wanting something in between 😀

I'm looking for a good suggestion right now...

**Comment by 0liver** [comment]
Maybe Lucida Grande?

![image](https://user-images.githubusercontent.com/164360/117355465-e5569300-aeb2-11eb-9238-369e6f5ac9b0.png)

**Comment by parisnoble** [comment]
Hi Everyone,
We've done a few adjustments to the display fonts. Please check.

It is important to know, that the wider the font we choose the wider the overall logo will become, which can limit the usage of the logo in digital.

We still believe that Oswald (the previous narrow font would work best), however, it is up to you to choose the final one.

Looking forward to hearing from you!


[Orchard core_rebranding_5.pdf](https://github.com/OrchardCMS/OrchardCore/files/6457529/Orchard.core_rebranding_5.pdf)

**Comment by 0liver** [comment]
### Here's the survey

https://freeonlinesurveys.com/s/IpDVCeSG

**Please go vote!**

**Comment by sebastienros** [comment]
@0liver where are the results?

**Comment by 0liver** [comment]
I would leave it up to @parisnoble call it an end and publish them.

**Comment by parisnoble** [comment]
Hi everyone,

and the winner is *drumrolls* Open Sans!

We will send all files ASAP!

Thanks for all the votes and feedback!

**Comment by vg** [comment]
Curious about the basis for the typeface choice? 

Other opensource typeface suggestions.

Selawik ( open source replacement for Microsoft Segoe UI)
https://github.com/microsoft/Selawik
https://www.fontsquirrel.com/fonts/selawik

**Comment by Piedone** [comment]
I'll soon submit a PR.
